consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/jsonrpc/methods/IbftGetPendingVotes.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.ibft.jsonrpc.methods;
+
+import tech.pegasys.pantheon.consensus.common.VoteProposer;
+import tech.pegasys.pantheon.consensus.common.jsonrpc.AbstractVoteProposerMethod;
+import tech.pegasys.pantheon.ethereum.jsonrpc.internal.methods.JsonRpcMethod;
+
+public class IbftGetPendingVotes extends AbstractVoteProposerMethod implements JsonRpcMethod {
+
+  public static final String PENDING_VOTES = "ibft_getPendingVotes";
```

Comment:
nit: should this be just called "JSON_RPC_NAME"? Then we could use the same thing everywhere ...

Reply:
Actually - I just saw that ibftGetValidatorsByBlockHash doesn't bother with the static member (i.e. the string name is just defined in the getName() function)... not sure which way I like more - but we should be consistent.